### PR TITLE
Cleanup of readme and creation of developing guide

### PR DESCRIPTION
**Pull request by dev-mlb**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -868,197 +266,12 @@ mkdir ~/Desktop/feed1
...
-
-### Running tests in IntelliJ throw an error "Error: Could not find or load main class @{argLine}"
-
-Did you read [the IntelliJ section](#intellij) carefully?
-
 ## Contact Us
 
+### General Questions
+
 If you have any questions or concerns about this project, you can contact us at: EmissarySupport@evoforge.org
```
Comment (burritoqueen) [inline README.md:273]: EmissarySupport@uwe.nsa.gov

File: DEVELOPING.md [hunk 2]
```diff
@@ -0,0 +1,706 @@
+# Developing Emissary
+
+Table of Contents
+=================
+* [Requirements](#requirements)
+* [IDE integration](#ide-integration)
+* [Coding Standards](#coding-standards)
+* [Internals](#internals)
```
Comment (burritoqueen) [inline DEVELOPING.md:8]: Reading through this, this felt out of place here. I personally like the old organization with the Internals first and then the "how to" with IDE/standards/commands/etc.

File: DEVELOPING.md [hunk 3]
```diff
@@ -0,0 +1,706 @@
...
+
+```
+PROJECT_BASE = ${project_loc}/target-eclipse
+```
+
+Unfortunately you will have to do this for every test, that's Eclipse with the M2E plugin for you.
+
+### Netbeans
+
+Unfortunately, I could not find a system property [Netbeans](https://netbeans.org/) sets when running
```
Comment (burritoqueen) [inline DEVELOPING.md:93]: I -> we to match everywhere else in the document

File: DEVELOPING.md [hunk 4]
```diff
@@ -0,0 +1,706 @@
...
+
+#### Transform
+
+The Transform phase of the workflow is expected to modify the bytes of the payload, set the current form to something
+new when appropriate, extract nested child payload objects, record metadata. After a Transform phase service place the
+workflow engine will go back to the ID phase and start evaluating for places that are interested in the currentForm.
+
+#### PostTransform
+
+Since the other primary phases of the workflow had "hook" stages in between them, it seemed
```
Comment (burritoqueen) [inline DEVELOPING.md:235]: Probably could remove this first sentence :)

File: DEVELOPING.md [hunk 5]
```diff
@@ -0,0 +1,706 @@
...
+### Potential Speed-up
+
+The above commands all have 'clean' in the command.  It is safest to do that.  But it is 
+possible to omit the 'clean' and maven will then only compile files that have changed.  This can
+speed up things, but for Emissary it is unlikely to speed up much.
+
+## Running Emissary
+
+There is one bash script in Emissary that runs everything.  It is in the top level Emissary directory.
+This script setups up a few environment variables and Java system properties.  It also sets up the
```
Comment (burritoqueen) [inline DEVELOPING.md:497]: setups up typo

File: README.md [hunk 6]
```diff
@@ -847,10 +245,10 @@ Run it with
 ./emissary topology
 ```
 
-### Running server with SSL
+#### Running server with SSL
 
 The keystore and keystore password are in the emissary.client.EmissaryClient-SSL.cfg file.  Included and configured
-by default is a sample keystore you can use for testing this functionality.  It is not recommended to use in 
+by default is a sample keystore you can use for testing this functionality -- it is not recommended using in
```
Comment (burritoqueen) [inline README.md:251]: still sounds like a grammar problem to me.  We do not recommend using? It is not recommended for use in?

File: README.md [hunk 7]
```diff
@@ -837,7 +235,7 @@ then goes to all the nodes it knows about and aggregates a collapsed view of the
 ./emissary pool --cluster
 ```
 
-### Topology (Clustered)
+#### Topology (Clustered)
 
 The topology talks to the configured node (localhost:8001 by default) and talks to every it knows about.
```
Comment (burritoqueen) [inline README.md:240]: talks to every _node_ it knows about

File: README.md [hunk 8]
```diff
@@ -828,7 +226,7 @@ information.  Run it like so:
 ./emissary agents --cluster
 ```
 
-### Pool (Cluster)
+#### Pool (Cluster)
 
 Pool in clustered mode also does the same pool in standalone.  It starts at the node (locahost:9001) by default
```
Comment (burritoqueen) [inline README.md:231]: does the same _as_ pool in standalone.

File: README.md [hunk 9]
```diff
@@ -682,21 +82,19 @@ arguments with descriptions for the *what* command, run:
...
-currently.
+Logging is handled by logback. You can point to a custom file with the *--logbackConfig* argument.
 
 See the *help -c <commandName>* for each command to get more info.
 
-### What
+#### What
 
 This command will use the configured engines to identify the file.  Emissary currently only comes with the 
 SizeIdPlace, so the id will TINY or SMALL etc.  See that class for more info.  The *-i or --input* argument
```
Comment (burritoqueen) [inline README.md:100]: id will _be_ TINY or SMALL

File: README.md (hunk 1 above)
Comment (dev-mlb) [reply README.md:273]: got it

File: DEVELOPING.md (hunk 2 above)
Comment (dev-mlb) [reply DEVELOPING.md:8]: I moved the section back to the top and renamed it overview.

**Review by jpdahlke — CHANGES_REQUESTED** [review]

**Comment by jpdahlke** [comment]
Near as I can tell, this does a good job of capturing the enhancements already made as part of the https://github.com/NationalSecurityAgency/emissary/pull/150 and subsequent review effort but reworks the file breakout.

**Review by jpdahlke — APPROVED** [review]

**Review by burritoqueen — APPROVED** [review]

**Review by jpdahlke — APPROVED** [review]

**Pull request merged**